transcendence-app/src/chat/chat.domain.ts

```diff
@@ -17,3 +18,15 @@ export class Chat {
     Object.assign(this, entity);
   }
 }
+
+export class ChatMember {
```

Comment:
praise: I like how you implement this class, looks good :+1:

suggestion: What do we think about moving it to a new file, `chatmember.domain.ts`?

Reply:
done